fileutil/fileutil.go

```diff
@@ -5,10 +5,38 @@
 package fileutil
 
 import (
+	"io/ioutil"
 	"os"
+	"path/filepath"
 	"sort"
+
+	"github.com/oklog/ulid"
 )
 
+func isBlockDir(fi os.FileInfo) bool {
+	if !fi.IsDir() {
+		return false
+	}
+	_, err := ulid.Parse(fi.Name())
+	return err == nil
+}
+
+// BlockDirs returns the list of block directories in the given directory.
+func BlockDirs(dir string) ([]string, error) {
```

Comment:
The fileutil package aims to provide helper functionality around file system primitives. I'm not sure this is the right place to add TSDB domain knowledge.

Reply:
I had kind of the same feeling when I put it here but didn't find any other package where it would fit. Should it be in its own util package?

Reply:
What's the incentive to move it out of the top-level package to begin with?